docs/outlook/one-outlook.md

```diff
@@ -0,0 +1,81 @@
+
+To help get you started on the migration process, review the following guidance.
+
+- The differences in features and scenarios supported by VSTO and COM add-ins and Outlook web add-ins are being addressed. To determine whether your add-in scenario is fully supported in an Outlook web add-in, see [Supported scenarios in Outlook web add-ins](#supported-scenarios-in-outlook-web-add-ins).
+- For guidance on how to transition your VSTO add-in to an Outlook web add-in, see [VSTO add-in developer's guide](../overview/learning-path-transition.md) and [Tutorial: Share code between both a VSTO Add-in and an Office Add-in with a shared code library](../tutorials/migrate-vsto-to-office-add-in-shared-code-library-tutorial.md).
+- If you're new to Outlook web add-ins, try out the [Outlook quick start](../quickstarts/outlook-quickstart.md) to build your first add-in.
+
+> [!NOTE]
+>
+> VSTO and COM add-ins are still supported in classic Outlook on Windows.
+
+### Supported scenarios in Outlook web add-ins
+
+The development of the Outlook JavaScript API used by Outlook web add-ins is focused on closing the gap on scenarios that are only supported by VSTO and COM add-in solutions. This way, users who transition to the Outlook web add-in can continue to have a seamless experience.
+
+The following table identifies key Outlook scenarios and their support status in a web add-in. This table will be updated as additional scenarios are supported. Periodically check this section as you plan to migrate your VSTO or COM add-in.
+
+|Scenario|Description|Support status in Outlook web add-ins|
+|-----|-----|-----|
+|Spam email reporting and education|Enable users to report unsolicited and potentially unsafe messages and learn how to identify these messages.|Supported. Improvements are in development to further enhance the user experience.|
+|Online meetings|Enable users to create and join online meetings.|Supported.|
+|Meeting enhancements|Provide additional services for users when they schedule meetings, such as location selection, catering services, and room lighting and temperature adjustments.|Supported.|
+|Online signatures|Automatically add themed signatures to messages and appointments.|Supported.|
+|Customer relationship management (CRM) and tracking services|Enable users to send and retrieve information from their CRM system to track communications with existing and potential customers.|Supported.|
+|Content reuse|Enable users to transfer and retrieve text and other content types from third-party systems.|Supported.|
```

Comment:
IIRC, instead of "third-party", we should use "partner" or similar term. @lindalu-MSFT, can you advise here?
```suggestion
|Content reuse|Enable users to transfer and retrieve text and other content types from partner systems.|Supported.|
```